src/styles/blocks/header.scss

```diff
@@ -0,0 +1,79 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  background-color: #fff;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.img {
+  height: 40px;
+  width: 40px;
+  display: block;
+}
+
+.nav {
+  display: block;
+}
+
+.nav__list {
```

Comment:
pls make use of nesting for all the `.nav` block, just like u did in `card.scss` file